server/api/api/utils.py

```diff
@@ -1343,6 +1352,27 @@ async def _delete_function(
     )
 
 
+async def _update_functions_with_deletion_task_ids(
+    db_session, functions, project, background_task_name
+):
+    semaphore = asyncio.Semaphore(
```

Comment:
I think it worth adding a log line with how many functions being updated, project name